plasmapy/formulary/collisions.py

```diff
@@ -1123,6 +1123,13 @@ def fundamental_electron_collision_freq(
     Average momentum relaxation rate for a slowly flowing Maxwellian
     distribution of electrons.
 
+    .. note::
```

Comment:
v_e is definitely too simple of a name for this. I think `Maxwellian_averaged_ei_collision_freq` is the shortest accurate name I think of